benchmarks/fmralign_benchopt/benchmark_meta.json

```diff
@@ -0,0 +1 @@
+{"pretty_name": "frmalign-benchopt"}
```

Comment:
Could you provide a nicer name for the benchmark? 😁 as is, it is hard to know what it is about.

Reply:
oops, it was actually supposed to be `fmralign-benchopt` not `frmalign-benchopt`. We'll fix and improve that. Thanks!

Reply:
Is ``brain-alignment-benchopt`` clearer ?

Reply:
This is the name that will appear on the front page of the results so you should be explicit.

I would say putting "fMRI alignment" is nice, without the bemchopt and the hyphen?

Note that this should be set as a `pretty_name` attribute in the `Objective` class.